fem/integ/bilininteg_diffusion_kernels.hpp

```diff
@@ -1051,33 +1051,33 @@ inline void SmemPADiffusionApply3D(const int NE,
       real_t (*QDD0)[MD1][MD1] = (real_t (*)[MD1][MD1]) (sm0+0);
       real_t (*QDD1)[MD1][MD1] = (real_t (*)[MD1][MD1]) (sm0+1);
       real_t (*QDD2)[MD1][MD1] = (real_t (*)[MD1][MD1]) (sm0+2);
-      MFEM_FOREACH_THREAD(dz,z,D1D)
+      MFEM_FOREACH_THREAD_DIRECT(dz,z,D1D)
```

Comment:
I believe this is always safe but is there a case where D1D > Q1D?

Reply:
Typically, we don't use that case, however, it may be good to add `MFEM_VERIFY` to error out if someone tries to use D1D > Q1D.

Reply:
done!